apps/web/src/modules/rejected/actor/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface EditActorInfoFormProps {
+  initialData: any
```

Comment:
any 🤔

Reply:
![image](https://user-images.githubusercontent.com/19553305/219969024-cc0ca567-11db-4699-a4ae-4b973296c598.png)
There is no DTO :ngae: